### ISSUE TITLE
Move from SPIFFS to LittleFS

### ISSUE THREAD

**Issue by OzHawk** [paulscottrobson/eris#30]
SPIFFS has been deprecated and is no longer actively supported.

LittleFS is currently being actively supported.  It also offers support for real directories and is many times faster than SPIFFS (not that its particularly important in this situation.

Move eris to LittleFS and implement new commands to support directories.

**Comment by OzHawk** [comment]
I'm not so sure on this one now.  ESP8266 says SPIFFS is deprecated, ESP32 doesn't mention LittleFS.

**Comment by paulscottrobson** [comment]
Wait and see. I think it's more important to add SD Card support, which isn't difficult I would think but I don't have a 1.4 board yet. (The reason there's no directories incidentally is SPIFFS doesn't support them)

**Comment by OzHawk** [comment]
Agreed...SD card support is more important.  I was going to ask you what hardware you had been testing on.  I currently have V1.4 of TTGO VGA32, and have also ordered Fabrizio's V4 Development board.  I'm also making up my own prototype board based on the schematic published by Fabrizio.

**Comment by paulscottrobson** [comment]
I have ordered some 1.4 SD cards but it'll be 2-3 weeks before they arrive. I'll have an optional build because there are three of these boards out there - they all have ps2 and vga, then they added the sound socket, then they added the SDcard connector. So it's worth keeping SPIFFS as an option.

**Comment by OzHawk** [comment]
From my research, the pins used for TTGO SD card are different from Fabrizio's Development board.  So it would be necessary to be able to change the pin designations fairly easily.
As for SD vs SPIFFS, is it possible to support both?  That way you could copy files from one to the other.
I believe the change from SPIFFS to LittleFS would be a replacement, but again, I didn't find evidence that SPIFFS is deprecated for ESP32.
I'm not sure how these file system changes we are discussing affect the other platforms supported by eris.
